.travis.yml

```diff
@@ -10,17 +10,20 @@ python:
 cache:
   pip: true
   directories:
-    - node_modules
+    - coalahtml/_coalahtml/node_modules
+    - coalahtml/_coalahtml/bower_components
 
 before_install:
   - sudo apt-get -y install python3-pip python-dev
   - sudo pip3 install -U pip setuptools virtualenvwrapper
 
   - npm install -g bower
+  - cd coalahtml/_coalahtml
   - bower install
+  - cd -
   - sudo pip3 install -r requirements.txt
 
-after_script:
+script:
```

Comment:
@AbdealiJK  Current master has `after_script` that makes travis to run `make test` as default. Is that what we wanted? 
I have changed it to `script:` now.